src/main/java/org/eaa690/aerie/communication/Message.java

```diff
@@ -0,0 +1,5 @@
+package org.eaa690.aerie.communication;
+
+public interface Message {
```

Comment:
Not sure this class is needed.  There are already `Message` classes in various packages/libraries.  Perhaps we can re-use one of those?